default.vcl

```diff
@@ -59,6 +59,9 @@ sub vcl_recv {
         set req.url = regsub(req.url, "notification\/brightcove\/metadata", "notify");
         set req.backend_hint = dynBackend.backend("brightcove-metadata-preprocessor");
     } 
+    elif (req.url ~ "\/schemas.*$") {
```

Comment:
Looking at the rest of them, half have start/beginning ^ and the rest don't have it while having the end of line anchor.

But since it's black regex magic, it's up to you to make it `^\/schemas.*$`

Reply:
:+1: